web/crux/assets/install-script/install-docker.sh.hbr

```diff
@@ -93,12 +93,12 @@ set_environment() {
     fi
   fi
 
-  if [ -z ${HOST_DOCKER_SOCK_PATH:-} ]; then
-    if [ -z ${DOCKER_HOST:-} ]; then
+  if [ -z "${HOST_DOCKER_SOCK_PATH:-}" ]; then
+    if [ -z "${DOCKER_HOST:-}" ]; then
       HOST_DOCKER_SOCK_PATH="/var/run/docker.sock"
     else
-      if [ ${DOCKER_HOST:0:7} = "unix://" ]; then
-        HOST_DOCKER_SOCK_PATH=${DOCKER_HOST:7}
+      if [ $(echo "$DOCKER_HOST" | cut -b -7) = "unix://" ]; then
+        HOST_DOCKER_SOCK_PATH="$DOCKER_HOST"
```

Comment:
@c3ppc3pp in the original code it copied the initials only with `${DOCKER_HOST:0:7}` was that intentional?

Reply:
Yeah, HOST_DOCKER_SOCK_PATH is just a path(iirc we use it for traefik deployment, but fix me if im wrong), while DOCKER_HOST is always should contain the protocol too.